src/_guides/language/sound-problems.md

```diff
@@ -20,7 +20,27 @@ and see [these other resources](/guides/language/type-system#other-resources).
 
 ## Troubleshooting
```

Comment:
It looks like it was this way before, but our style guidelines want at least one sentence between headers.